### ISSUE TITLE
An example for authenticated websocket session

### ISSUE THREAD

**Issue by bvk** [preichenberger/go-gdax#52]
There is now a Headers function defined to authenticate websocket sessions. It would be helpful to add an example to the README file as well. Perhaps, a websocket session example to subscribe to user channel would be useful.

@arussellsaw

**Comment by forward32** [comment]
It is very easy to implement by yourself, for example you can look how I did it (https://github.com/forward32/go-gdax/commit/fad5d77ffde30c9fa2e2ae42f0660dd6835023a0).

**Comment by bvk** [comment]
I agree.  The older version I had locally didn't have SignedMessage type, so it wasn't clear. Closing the ticket now.
